packages/client-app/src/libs/sendRequest.ts

```diff
@@ -36,26 +36,34 @@ export const sendRequest = async (
     }
   })
 
+  const headers = paramsReducer(instance.parameters.headers)
+
   let data: FormData | string | File | null = null
 
   if (instance.body.activeBody === 'binary' && instance.body.binary) {
-    data = instance.body.binary
+    headers['Content-Type'] = 'multipart/form-data'
+
+    const bodyFormData = new FormData()
+    bodyFormData.append(instance.body.binary.name, instance.body.binary)
+    data = bodyFormData
```

Comment:
If the body type is binary then the fetch body is directly a blob. 

Multipart form can also be used for files but the structure is different and you can include mixed data (files and string). 

From a UX perspective if binary is selected there should just be a single file upload box and nothing else.

For multipart form we should show the DataTable and each value cell can be a file OR a string. 

It matter quite a bit with regards to importing spec definitions because servers have to handle multipart form data very differently.